scripts/uninstall.sh

```diff
@@ -37,55 +39,63 @@ fi
 
 if [ ! -f $FUNCTION_ACTIVATION_CONFIG ]; then
     echo -e "INFO: Configuration file [$FUNCTION_ACTIVATION_CONFIG] missing, extracting default from release"
-    wget -q $FUNCTION_REPOSITORY_RELEASE_URL -O $WORKING_DIR/$FUNCTION_ZIP_PACKAGE
-    unzip -o -q $WORKING_DIR/$FUNCTION_ZIP_PACKAGE -d $TMP_FUNCTION_DIR || exit
-    mv $TMP_FUNCTION_DIR/$FUNCTION_ACTIVATION_CONFIG $FUNCTION_ACTIVATION_CONFIG
+    wget -q "$FUNCTION_REPOSITORY_RELEASE_URL" -O "$WORKING_DIR"/$FUNCTION_ZIP_PACKAGE
+    unzip -o -q "$WORKING_DIR"/$FUNCTION_ZIP_PACKAGE -d "$TMP_FUNCTION_DIR" || exit
+    mv "$TMP_FUNCTION_DIR"/$FUNCTION_ACTIVATION_CONFIG $FUNCTION_ACTIVATION_CONFIG
     echo
 fi
 
-readonly GCP_PROJECT=$("$YQ" e '.googleCloud.required.gcpProjectId' $FUNCTION_ACTIVATION_CONFIG)
-readonly GCP_SERVICE_ACCOUNT=$("$YQ" e '.googleCloud.common.serviceAccount' $FUNCTION_ACTIVATION_CONFIG)
-readonly GCP_PUBSUB_TOPIC=$("$YQ" e  '.googleCloud.metrics.pubSubTopic' $FUNCTION_ACTIVATION_CONFIG)
-readonly GCP_FUNCTION_NAME=$("$YQ" e '.googleCloud.metrics.function' $FUNCTION_ACTIVATION_CONFIG)
-readonly GCP_SCHEDULER_NAME=$("$YQ" e '.googleCloud.metrics.scheduler' $FUNCTION_ACTIVATION_CONFIG)
-readonly DYNATRACE_URL_SECRET_NAME=$("$YQ" e '.googleCloud.common.dynatraceUrlSecretName' $FUNCTION_ACTIVATION_CONFIG)
-readonly DYNATRACE_ACCESS_KEY_SECRET_NAME=$("$YQ" e '.googleCloud.common.dynatraceAccessKeySecretName' $FUNCTION_ACTIVATION_CONFIG)
+GCP_PROJECT=$("$YQ" e '.googleCloud.required.gcpProjectId' $FUNCTION_ACTIVATION_CONFIG)
```

Comment:
Does this readonly split give us any value?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
this that we will 100% sure what is inside and it is readonly, if I understand good if something goes wrong correct value an't be write to this variable